.github/workflows/multi-platform-release.yml

```
@@ -0,0 +1,27 @@
+
+name: Extension Release
+
+on: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)